.github/workflows/container-image.yml

```diff
@@ -46,6 +46,7 @@ jobs:
           push: true
           secrets: |
             NODE_AUTH_TOKEN=${{ secrets.PERSONAL_ACCESS_TOKEN }}
+            GIT_AUTH_TOKEN=${{ secrets.PERSONAL_ACCESS_TOKEN }}
```

Comment:
Do we need this now once the repository is public?

Reply:
No, reverted.